README.md

```diff
@@ -61,6 +61,31 @@ pub struct Rgb {
 }
 ```
 
+### Customizing single fields
+
+This can be particular handy if your structure uses a type that does not implement `Arbitrary` or you want to have more customization for particular fields.
+
+```rust
+#[derive(Arbitrary)]
+pub struct Rgb {
+    // set `r` to Default::default()
+    #[arbitrary(default)]
+    pub r: u8,
+
+    // set `g` to 255
+    #[arbitrary(value = "255")]
+    pub g: u8,
+
+    // generate `b` with a custom function
+    #[arbitrary(with = "arbitrary_b")]
```

Comment:
Similarly, I think this should be an ident or path rather than a string, and we can implement this by just passing the RHS tokens through rather than parsing them. For example:

```rust
#[arbitrary(with = super::foo::bar::my_func)]
```

Reply:
Addressed in https://github.com/rust-fuzz/arbitrary/pull/129/commits/f73d5f2333f4004ccdd5250c198044e5102d61f9